src/js/actions/search/libraries.js

```diff
@@ -144,15 +144,15 @@ define(function (require, exports) {
                 selectPromise = this.flux.actions.libraries.selectLibrary(asset.library.id);
 
             switch (elementInfo.type) {
-                case "GRAPHIC":
-                    var uiStore = this.flux.stores.ui,
-                        panelStore = this.flux.stores.panel,
-                        centerOffsets = panelStore.getState().centerOffsets,
-                        midX = (window.document.body.clientWidth + centerOffsets.left - centerOffsets.right) / 2,
-                        midY = (window.document.body.clientHeight + centerOffsets.top - centerOffsets.bottom) / 2,
-                        location = uiStore.transformWindowToCanvas(midX, midY);
+            case "GRAPHIC":
+                var uiStore = this.flux.stores.ui,
+                    panelStore = this.flux.stores.panel,
+                    centerOffsets = panelStore.getState().centerOffsets,
+                    midX = (window.document.body.clientWidth + centerOffsets.left - centerOffsets.right) / 2,
+                    midY = (window.document.body.clientHeight + centerOffsets.top - centerOffsets.bottom) / 2,
+                    location = uiStore.transformWindowToCanvas(midX, midY);
                     
-                    selectPromise
+                selectPromise
                         .bind(this)
```

Comment:
Maybe we should remove the extra indents below as well? Wondering why Eslint is not reporting it.